user/src/main/java/org/the_chance/honeymart/ui/feature/home/composables/ProductItem.kt

```diff
@@ -25,17 +25,16 @@ import org.the_chance.honymart.ui.theme.dimens
 import org.the_chance.honymart.ui.theme.white
 
 @Composable
-fun NewProductsItems(
+fun ProductItem(
     modifier: Modifier = Modifier,
```

Comment:
the modifier parameter should be the first parameter with the default value, to be like this : 
isFavoriteIconClicked: Boolean,
onClickFavorite: () -> Unit,
productName: String,
productPrice: String,
modifier: Modifier = Modifier,
enable: Boolean = true,